apps/admin/backend/src/app.diagnostics.test.ts

```diff
@@ -44,48 +45,52 @@ test('diagnostic records', async () => {
 
   jest.setSystemTime(new Date(1000));
   await apiClient.addDiagnosticRecord({
-    hardware: 'printer',
+    type: 'test-print',
     outcome: 'fail',
   });
-  expect(await apiClient.getDiagnosticRecords()).toEqual([
-    {
-      hardware: 'printer',
-      outcome: 'fail',
-      timestamp: 1000,
-    },
-  ]);
+  expect(await apiClient.getDiagnosticRecords()).toEqual(
+    typedAs<DiagnosticRecord[]>([
```

Comment:
no need to change -- just a heads-up you could also do `expect().toEqual<DiagnosticRecord[]>()` to avoid having to use `typedAs`

Reply:
omg, exactly what I want